src/frontend/src/icp/components/fee/EthereumEstimatedFee.svelte

```diff
@@ -46,6 +46,11 @@
 			? maxTransactionFeePlusEthLedgerApprove + CKERC20_TO_ERC20_MAX_TRANSACTION_FEE
 			: maxTransactionFeePlusEthLedgerApprove;
 
+	let feeSymbol: string;
+	$: feeSymbol = ckEr20
+		? tokenCkEth?.symbol ?? $ckEthereumNativeToken.symbol
```

Comment:
is it ok to default to `ckEthereumNativeToken` ? conceptually, if `tokenCkEth` is `nullish`, shouldn't we raise an error? can it be even possible to be `nullish` if `ckEr20` is true?

Reply:
Conceptually agree it's an error but, 1 ckEth === 1 Ethereum, so actually displaying with `ckEthereumNativeToken` isn't wrong neither. Therefore I thought that preventing this edge case this way felt acceptable.